.expeditor/website.pipeline.yml

```diff
@@ -1,16 +1,27 @@
+expeditor:
+  secrets:
+    FASTLY_API_KEY:
+      path: account/static/fastly/eng-services-ops
+      field: token
+  accounts:
+    - aws/chef-cd
+    - aws/habitat
+  defaults:
+    buildkite:
+      timeout_in_minutes: 30
+      env:
+        FASTLY_SERVICE_KEY: "T32H9RqMWpCV9qhp3S9xq"
```

Comment:
There's a different service key baked into ` .expeditor/scripts/purge_cdn.sh`... is that intentional? If so, what's the distinction between them?

Reply:
The `purge_cdn.sh` script is purging the cache for `packages.chef.io`.  That script isn't part of the work on this PR, I just updated it to use the new function.  The service key above is for `www.habitat.sh`.

Reply:
It'd be a good idea to add some explanatory comments to both of these places to indicate which service the keys pertain to, just to make it easier to reason about.

Reply:
Ok

Reply:
Fixed in https://github.com/habitat-sh/habitat/pull/7273/commits/7eecaa4a2b60b1610076b0c703b06df1cdd2ec6c